### PR TITLE
Webhooks docs update

### PR DESCRIPTION
**Pull request by SleepyLili**
When working on my bot, I noticed the webhooks docs were a little outdated.

I updated the example jsons to include new fields that they now have, fixed their indentation, and included a new `VISIBILITY-UPDATE` example since the json is now completely different from the old docs.

I also did some updates on the wording on the page -- tried to fix capitalization/punctuation mistakes, and then update the sentences to be more readable. I'm not 100% happy with all the changes I made, so they don't need to be included if you don't like them. (Or if anyone has an idea on how to further improve them...)

### CODE REVIEW

File: content/thirdparty/webhooks.md [hunk 1]
```diff
@@ -66,10 +68,14 @@ where
             "displayName": "lazyDaisy",
             "displayColor": 182,
             "createdAt": "2021-08-12T07:51:37.470812684Z",
-            "previousNames": ["lazyDaisy"]
+            "previousNames": ["lazyDaisy"],
+            "nameChangedAt": "2022-09-19T12:33:59.42313245+02:00",
+            "isBot": False,
```
Comment (gabek) [inline content/thirdparty/webhooks.md:73]: Not sure how the f in `False` got capitalized, but could you fix that up for all of them?

**Comment by gabek** [comment]
Yeah, it looks good aside from the rogue capitals in the JSON examples.

I find the changes to be completely readable, thank you for taking some time to polish it up! This kind of stuff really makes a difference when people are trying to implement APIs, as I'm sure you discovered when doing it yourself :)

**Comment by SleepyLili** [comment]
good catch with the capitalization, fixed it.

**Comment by gabek** [comment]
Thank you so much! 🥇

**Pull request merged**